src/components/scenegraph/Toolbar.js

```diff
@@ -69,13 +82,42 @@ export default class Toolbar extends React.Component {
    * Try to write changes with aframe-inspector-watcher.
    */
   writeChanges = () => {
-    const xhr = new XMLHttpRequest();
-    xhr.open('POST', 'http://localhost:51234/save');
-    xhr.onerror = () => {
-      alert('aframe-watcher not running. This feature requires a companion service running locally. npm install aframe-watcher to save changes back to file. Read more at supermedium.com/aframe-watcher');
-    };
-    xhr.setRequestHeader('Content-Type', 'application/json');
-    xhr.send(JSON.stringify(AFRAME.INSPECTOR.history.updates));
+    const baseUrl = "http://localhost:8888/";
```

Comment:
Is it possible to use like an env variable or config variable or something here? Just so we don't have to modify it in multiple places when we are switching to prod. I am not sure if such a thing exists in React, ignore this comment if it doesn't.

Reply:
There is -- see https://github.com/uwblueprint/cdc-frontend/pull/28/files for how I did it in a different PR.

Stack overflow question: https://stackoverflow.com/questions/49579028/adding-an-env-file-to-react-project